networking/changing-cluster-network-mtu.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="changing-cluster-network-mtu"]
+= Changing the MTU for the cluster network
+include::modules/common-attributes.adoc[]
+:context: changing-cluster-network-mtu
+
+toc::[]
+
+[role="_abstract"]
+As a cluster administrator, you can change the MTU for the cluster network after cluster installation. This change is disruptive as cluster nodes must be rebooted to finalize the MTU change. You can change the MTU only for clusters using the OVN-Kubernetes or OpenShift SDN cluster network providers.
+
+include::modules/nw-cluster-mtu-change-about.adoc[leveloffset=+1]
+include::modules/nw-cluster-mtu-change.adoc[leveloffset=+1]
+
+[id="{context}-additional-resources"]
+== Additional resources
+
+* xref:../installing/installing_bare_metal/installing-bare-metal.adoc#installation-user-infra-machines-advanced_network_installing-bare-metal[Using advanced networking options for PXE and ISO installations]
+* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_networking/assembly_manually-creating-networkmanager-profiles-in-key-file-format_configuring-and-managing-networking[Manually creating NetworkManager profiles in key file format]
```

Comment:
My only concern with links on Line 19 and Line 20 is that we take our customers out of their doc experience and put them on the portal. This isn't so bad for customers viewing their docs on the portal but if they are viewing it on docs.openshift.com they're being redirected to a different user experience. Is there a way to get to this content another way? If not, then ignore this comment.

Reply:
@sagidlow, true, but the RHEL docs are available only on the customer portal.

Reply:
@jboxman-rh that's what I figured but thought I would voice my concern anyway :)